components/openindiana/docs/Makefile

```diff
@@ -0,0 +1,69 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016, Aurelien Larcher
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=     oi-docs
+COMPONENT_FMRI=     release/openindiana-docs
+COMPONENT_SUMMARY=  OpenIndiana Docs - Online Documentation
+COMPONENT_CLASSIFICATION= Desktop (GNOME)/Documentation
+COMPONENT_PROJECT_URL= https://github.com/OpenIndiana/oi-docs
+COMPONENT_ARCHIVE_URL= https://github.com/OpenIndiana/oi-docs
+COMPONENT_SRC=      $(COMPONENT_NAME)
+COMPONENT_LICENSE= PDL
+
+GIT_REPO=git://github.com/OpenIndiana/oi-docs.git
+GIT_BRANCH=master
+GIT_CHANGESET=HEAD
+
+include prep-git.mk
```

Comment:
Well, I see two ways: 1) make prep-git.mk common for several components which rely on fetching sources from git/github, 2) make this component more like existing components which already checkout sources from github. Perhaps, we even don't want to checkout latest sources, just exact revision (and just can fetch it as usual archive).